public_html/templates/base.html

```diff
@@ -75,25 +77,39 @@
 
 	{% block main %}
 		<section class="filters container-fluid">
-			<form>
-				<fieldset class="form-group filters-fieldset text-center">
-					{% for type, description in filterTypes %}
-						<span class="radio-inline">
+			<form id="filters-form">
+				<fieldset class="form-group filters-fieldset col-lg-6">
```

Comment:
Your phone is huge!!!! :) I guess there actually is a grid class on the select box, but I don't think we need them at all. I'll push back some updates shortly, and also make them centered.